app/src/main/java/org/fossasia/openevent/general/search/SearchLocationFragment.kt

```diff
@@ -36,62 +43,38 @@ class SearchLocationFragment : Fragment() {
     private val safeArgs: SearchLocationFragmentArgs by navArgs()
     private val eventLocationList: MutableList<String> = ArrayList()
 
+    private val placeSuggestionsAdapter = PlaceSuggestionsAdapter(PlaceDiffCallback())
```

Comment:
No need to pass in as constructor. Just make it a property in the adapter itself

Reply:
done